src/monster.cpp

```diff
@@ -3126,6 +3159,44 @@ void monster::process_one_effect( effect &it, bool is_new )
             you.get_sick( true );
         }
     }
+
+    //Process enchantments that apply to monsters.
+
+    //These maps exist to mark enchantment mods we've seen once before. That way, if we get a collision, we can add them together instead of overwriting them.
+    std::map<const enchant_vals::mod, bool> values_add_exist;
+    std::map<const enchant_vals::mod, bool> values_mult_exist;
+    for( const auto &elem : *effects ) {
```

Comment:
This feels sloppy to me - If I knew how to make a blank enchantment, I could force add them together and then go through one mod list instead of all of them.